src/test/csf/ledgers.h

```diff
@@ -137,7 +143,13 @@ class Ledger
     }
 
 public:
-    Ledger() : id_{0}, instance_(&genesis)
+    Ledger(MakeGenesis) : instance_(&genesis)
+    {
+    }
+
+    // This is required by the generic Consensus for now and should be
+    // migrated to the MakeGenesis approach above.
+    Ledger() : Ledger(MakeGenesis{})
```

Comment:
@seelabs I liked your suggestion for making the tagged constructor, but generic `Consensus` still expects a default constructor. I would like to fix that, but would prefer to do it as a separate refactor, so this is needed in the interim.